_posts/platform/cli/2000-01-01-features.md

```diff
@@ -18,12 +18,17 @@ the CLI requires you to take one of these actions:
 ## Tips
 
 For all the following subcommands:
-* You can use the environment variable `SCALINGO_APP` instead of using the `--app` flag
-* If your current directory is the base directory of your project
-  and that your Git repository has a remote named `scalingo`, you
-  don't need to specify `--app <name>`. It will be detected automatically.
+
+* You can use the environment variable `SCALINGO_APP` instead of using the
+  `--app` flag
+* If your current directory is the base directory of your project and that your
+  Git repository has a remote named `scalingo`, you don't need to specify `--app
+  <name>`. It will be detected automatically.
 * If you want to specify a different remote name, you can do it by using
-  `--remote` or `-r` flag followed by the name
+  `--remote` or `-r` flag followed by the name.
+* If you connect to the internet through a proxy, define the environment
+  variables `http_proxy` and `https_proxy` with your proxy hostname and port
```

Comment:
Lower case variable name ?

Reply:
BTW it should be noted that it's a standard mechanism on Linux system (maybe MacOS X as well?)

Reply:
> Lower case variable name ?

Yep as noted here (https://github.com/Scalingo/cli/#run-behind-a-proxy). After further investigation, both lower and upper case work. I update to display the upper case as I prefer it.